whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/SerialRpslObjectTuple.java

```diff
@@ -0,0 +1,29 @@
+package net.ripe.db.whois.common.dao.jdbc;
+
+import net.ripe.db.whois.common.dao.RpslObjectModel;
+import net.ripe.db.whois.common.dao.Serial;
+
+
+public class SerialRpslObjectTuple {
+
+    private final Serial serial;
+    private final RpslObjectModel rpslObjectModel;
```

Comment:
I think what you are looking for is SerialEntry bean. Please go through existing NRTM code , we have almost all he beans we need. I am not sure why SerialEntry does not contain serial_id but please feel free to add it there as its already available. Let's discuss over call

Reply:
Already changed in new commit